src/argus_htmx/templates/htmx/incidents/_table.html

```diff
@@ -0,0 +1,89 @@
+<table id="table">
+    {% block table %}
+        <thead>
+        <tr>
+            <th>actions</th>
+            <th>id</th>
+            <th>timestamp</th>
+            <th>status</th>
+            <th>severity</th>
+            <th>source</th>
+            <th>description</th>
+            <th>ack</th>
+            <th>tag</th>
```

Comment:
These need to be swappable. New block "incident_table_header" covering all of them or something..